velox/exec/Task.h

```diff
@@ -263,7 +263,10 @@ class Task : public std::enable_shared_from_this<Task> {
   };
 
   /// Collect long running operator calls across all drivers in this task.
-  void getLongRunningOpCalls(
+  /// Return false when the lock cannot be taken within the timeout, in that
+  /// case the result is not populated.  Return true if everything works well.
+  bool getLongRunningOpCalls(
+      std::chrono::nanoseconds lockTimeout,
```

Comment:
What is the timeout? Can't find where is it specific for some reason.

Reply:
It's the timeout for locking the per task mutex.  I will update the comment to make it more explicit.

Reply:
But what value do we use for this timeout? Where can I see that?

Reply:
It will be in [D52921877](https://www.internalfb.com/diff/D52921877) but that is still work in progress